wazo_calld/plugin_helpers/ari_.py

```diff
@@ -73,6 +76,31 @@ def get_value():
     raise Exception(f'failed to set channel variable {var}={value}')
 
 
+T = TypeVar('T')
+
+
+class GlobalVariableAdapterProtocol(Protocol[T]):
```

Comment:
Global variables in Asterisk must be strings, no need for abstract types here.

Reply:
The need for generic types is to support the JSON adapter.

Reply:
I see, thanks.